Week3/homework-midy/removeDuplicates.js

```diff
@@ -0,0 +1,7 @@
+const letters = ['a', 'b', 'c', 'd', 'a', 'e', 'f', 'c', 'b'];
+
+function removeDuplicates(array) {
+  return [...new Set(array)];
+}
```

Comment:
Cool way that gets the job done. If this was a real project I would say kudos and accept this solution. 
But for the purpose of this week's exercise you need to achieve the same using closures. Similar to what we did in the class

Reply:
Alright I'll try that!